### DISCUSSION TITLE
looooooooooove DaisyUI

### DISCUSSION THREAD

**Discussion by lindamacdonald** [saadeghi/daisyui#438] [General]
just sayin' you guys are awesome :) I was wondering how I use a different theme than the default in a nextjs project?

**Comment by saadeghi** [comment]
@lindamacdonald 
- [How to use themes](https://daisyui.com/docs/default-themes)
- [How to add new theme](https://daisyui.com/docs/add-themes)

**Reply by lindamacdonald** [reply]
Hey again :) I did actually browse through your docs but dont quite understand where it all fits in in the tailwind config for next. could you please advise? tried using instructions on daisui themes in extends but that doesnt work. could you please let me know, from the below config where and how I should use the daisyui theme?
module.exports = {
  content: [
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",
  ],
  theme: {
    extend: {},
  },
  plugins: [

    require('daisyui')
  ],
}

**Reply by saadeghi** [reply]
let's continue the conversation in one place: https://github.com/saadeghi/daisyui/issues/439

**Reply by lindamacdonald** [reply]
I am not allowed to comment on the other one so cannot continue in the other thread
